config/samples/custom-theme/README.md

```diff
@@ -0,0 +1,274 @@
+# Customizing Horizon Dashboard Theme with horizon-operator
+
+The horizon-operator provides mechanisms to customize the Horizon Dashboard
+theme. A theme is a directory containing assets that override the default
+styling of the dashboard. Horizon can be configured to support multiple themes
+simultaneously, making them available at runtime. Through the horizon-operator,
+you can load custom themes using the
+[ExtraMounts](https://github.com/openstack-k8s-operators/dev-docs/blob/main/extra_mounts.md)
+feature.
+
+## Theme configuration
+
+To provide additional themes, you must specify them in the `AVAILABLE_THEMES`
+setting within the `_11_rcue_theme.py` file. Themes are defined as a list of
+tuples with the following format:
+
+```
+('name', 'label', 'path')
+```
+
+Where:
+
+- name: The key by which the theme value is resolved
+- label: The display name or description of the custom theme
+- path: The directory location where the theme is unpacked and stored
+
+Currently, the `horizon-operator` only supports the default path:
+`/usr/share/openstack-dashboard/openstack_dashboard/themes`
+
+## Procedure
+
+To use a custom theme, create a `_11_rcue_theme.py` file and configure the
+`AVAILABLE_THEMES` as follows:
+
+```python
+# override the CUSTOM_THEME_PATH variable with this settings snippet
+AVAILABLE_THEMES = [('rcue', 'Custom Theme', 'themes/rcue')]
```

Comment:
As I was reading this I wondered what the relative path `themes/rcue` would be expanded to. I found it with some quick googling "The path must be relative to the openstack_dashboard directory" so I assume it's going under `/usr/share/openstack-dashboard/openstack_dashboard/themes`.

Do you want to add:
```
See the [Horizon Theme Documentation](https://docs.openstack.org/horizon/latest/configuration/themes.html) for more details.
```

Reply:
Thank you John, added!